src/Raven.Server/Documents/Handlers/StreamCsvResultWriter.cs

```diff
@@ -29,7 +29,10 @@ public abstract class StreamCsvResultWriter<T> : IStreamQueryResultWriter<T>
         {
             Constants.Documents.Metadata.Attachments,
             Constants.Documents.Metadata.Counters,
-            Constants.Documents.Metadata.Flags
+            Constants.Documents.Metadata.Flags,
+            Constants.Documents.Metadata.ChangeVector,
+            Constants.Documents.Metadata.Id,
+            Constants.Documents.Metadata.LastModified
```

Comment:
We do we need to add additional properties to skip?

Reply:
Yes, added according to https://github.com/ravendb/ravendb/blob/dd3827a368291ccbb37c6af37836e6de98557b87/src/Raven.Server/Documents/DocumentPutAction.cs#L800